builder running on an old centos (version 7)

Hi,

I noticed that the centos builder uses centos7, while centos8 has been out for several months (according to wikipedia September 2019). Is there a good reason for that, or could the centos builder be upgraded to centos 8?

Thanks.

Additionally, I tried to mark a package not available on centos 7 (unfortunately opam only let's me express this via `available: os-distribution != "centos"` which marks it as unavailable for centos, independent of version), and the CI reported an error in the builder (where I expected a pass / warning -- similar to if a package has lower/upper bounds on the OCaml version). The CI output:
```
#19 [stage-0 12/12] RUN --mount=type=cache,target=/home/opam/.opam/download-...
#19 1.104 mirage-crypto is now pinned to git+file:///src#master (version ~dev)
#19 1.106 mirage-crypto-rng is now pinned to git+file:///src#master (version ~dev)
#19 1.108 mirage-crypto-pk is now pinned to git+file:///src#master (version ~dev)
#19 1.599 [ERROR] mirage-crypto-rng = ~dev unmet availability conditions: os-distribution != "centos"
#19 1.600 'opam install -tv .' failed.
#19 ERROR: executor failed running [/bin/sh -c opam install -tv .]: runc did not terminate sucessfully
```

(from https://ci.ocamllabs.io/github/mirage/mirage-crypto/commit/1fa0105e70c74c0422584d5b9fbd704bac9fedfd/variant/centos)

--- a/service/pipeline.ml
+++ b/service/pipeline.ml
@@ -65,7 +65,7 @@ let build_with_docker ~repo ~analysis source =
     "alpine", build (module Conf.Builder_amd1) @@ "alpine-3.10-ocaml-" ^ default_compiler;
     "ubuntu", build (module Conf.Builder_amd2) @@ "ubuntu-19.04-ocaml-" ^ default_compiler;
     "opensuse", build (module Conf.Builder_amd2) @@ "opensuse-15.1-ocaml-" ^ default_compiler;
-    "centos", build (module Conf.Builder_amd3) @@ "centos-7-ocaml-" ^ default_compiler;
+    "centos", build (module Conf.Builder_amd3) @@ "centos-8-ocaml-" ^ default_compiler;
     "fedora", build (module Conf.Builder_amd3) @@ "fedora-30-ocaml-" ^ default_compiler;
     (* oraclelinux doesn't work in opam 2 yet: *)
     (* build (module Conf.Builder_amd3) @@ "oraclelinux-7-ocaml-" ^ default_compiler; *)

